mo

```diff
@@ -21,7 +21,9 @@
 #
 # $0 - Name of the mo file, used for getting the help message.
 # $* - Filenames to parse.  Can use -h or --help as the only option
-#      in order to show a help message.
+#      in order to show a help message. Additionaly, --source=file
+#      can be passed so that mo sources the file before parsing
```

Comment:
The instructions do not make it clear that the options must come first, before any filenames.  Also, we should support `--` to indicate that no further options should be parsed.  I'm happy to make that change if you're not comfortable doing it.  That way I could have a template file called `--source=template` and I can still get `mo` to process it.

I can add the `--` support if necessary.